common/frame_session.go

```diff
@@ -616,16 +617,13 @@ func (fs *FrameSession) onConsoleAPICalled(event *cdpruntime.EventConsoleAPICall
 		}
 		*/
 
-	parsedObjects := make([]any, 0, len(event.Args))
+	parsedObjects := make([]string, 0, len(event.Args))
 	for _, robj := range event.Args {
-		i, err := parseRemoteObject(robj)
-		if err != nil {
-			handleParseRemoteObjectErr(fs.ctx, err, l)
-		}
+		i := parseConsoleRemoteObject(fs.logger, robj)
 		parsedObjects = append(parsedObjects, i)
```

Comment:
IDK why it's been done so with `i` before (probably to say it's an interface)—it feels like we're getting an index. I believe it's better to name this `s` because the returned value is a `string`.

```suggestion
		s := parseConsoleRemoteObject(fs.logger, robj)
		parsedObjects = append(parsedObjects, s)		
```